### ISSUE TITLE
Unhandled exception (#a7bc2809)

### ISSUE THREAD

**Issue by sqlmapreporter** [sqlmapproject/sqlmap#2570]
```
sqlmap version: 1.0.9.38#dev
Python version: 2.7
Operating system: nt
Command line: sqlmap.py -u *********************************** --forms -p username --time-sec 10 --level 2 --risk 2 --tamper charencode -D ***************** --tables
Technique: TIME
Back-end DBMS: MySQL (fingerprinted)
```
```
Traceback (most recent call last):
  File "sqlmap.py", line 149, in main
    start()
  File "lib/controller/controller.py", line 633, in start
    action()
  File "lib/controller/action.py", line 103, in action
    conf.dumper.dbTables(conf.dbmsHandler.getTables())
  File "plugins/generic/databases.py", line 313, in getTables
    count = inject.getValue(query, union=False, error=False, expected=EXPECTED.INT, charsetType=CHARSET_TYPE.DIGITS)
  File "lib/request/inject.py", line 445, in getValue
    value = _goInferenceProxy(query, fromUser, batch, unpack, charsetType, firstChar, lastChar, dump)
  File "lib/request/inject.py", line 285, in _goInferenceProxy
    outputs = _goInferenceFields(expression, expressionFields, expressionFieldsList, payload, charsetType=charsetType, firstChar=firstChar, lastChar=lastChar, dump=dump)
  File "lib/request/inject.py", line 128, in _goInferenceFields
    output = _goInference(payload, expressionReplaced, charsetType, firstChar, lastChar, dump, field)
  File "lib/request/inject.py", line 100, in _goInference
    count, value = bisection(payload, expression, length, charsetType, firstChar, lastChar, dump)
  File "lib/techniques/blind/inference.py", line 593, in bisection
    val = getChar(index, asciiTbl)
  File "lib/techniques/blind/inference.py", line 397, in getChar
    threadData.validationRun += 1
AttributeError: '_ThreadData' object has no attribute 'validationRun'
```

**Comment by klensy** [comment]
[validationRun](https://github.com/sqlmapproject/sqlmap/commit/212c1ec1f248c277cac521240884b6b5380a54cf) added in 1.0.9.41, perhaps fixed.

**Comment by stamparm** [comment]
Duplicate of #2233 (fixed half a year ago with 1.0.9.41, as stated by @klensy)
